### PR TITLE
again  changes are done in pomc classes

### PR DESCRIPTION
**Pull request by veera977**
can u review my changes as soon as possible

### CODE REVIEW

File: Automation_FrameWORK2_dummy/src/main/java/Com/comcast/crm/ObjectrepositoryUtility/class1.java [hunk 1]
```diff
@@ -8,6 +8,11 @@ public static void main(String[] args) {
 		System.out.println("bye");
 		
 		System.out.println("thinnaraaa");
+		
+		
+		System.out.println("em thinnaru");
+		System.out.println("bagunnara");
```
Comment (veera977) [inline Automation_FrameWORK2_dummy/src/main/java/Com/comcast/crm/ObjectrepositoryUtility/class1.java:14]: review team lead

**Review by veera977 — COMMENTED** [review]
do earle

**Comment by Asha306** [comment]
use id locadtor
do not hard code the  data in testscripts

**Comment by veera977** [comment]
ok team lead i will send after doing

**Comment by Asha306** [comment]
merged successfully no need to change again

**Pull request merged**